content/applications/sales/sales/products_prices/ewallets_giftcards.rst

```diff
@@ -0,0 +1,143 @@
+Settings --> Pricing section`. Once enabled, go to :menuselection:`Sales app --> Products --> Gift
+cards & eWallet` and :guilabel:`Create` a new eWallet or gift card program.
+
+eWallets
+========
+
+eWallets allow customers to save credits on their online account and use these credits as a payment
+method when buying items in an online store or a brick-and-mortar store. eWallets can also be used
+to centralize multiple :ref:`gift cards <ewallet_gift/gift-cards>`.
+
+Before creating an eWallet program, it is necessary to create an eWallet **top-up** product.
+Top-ups are pre-defined digital credit values added to an eWallet in exchange for its equivalent in
+real currency. These credits can then be used as a payment method in the eCommerce shop or
+:abbr:`PoS (Point of Sale)`. Top-up values can be of different amounts.
+
+.. example::
+   A $50 top-up can be bought for $50, and adds that same amount of credits to the eWallet.
+
+To create a top-up product, go to :menuselection:`Sales app --> Products --> Products` and
+:guilabel:`Create` a new product. On the product template, configure the options as follows:
+
+- :guilabel:`Product Name`: enter a name for the top-up product (for example, `$50 Top-Up`)
+- :guilabel:`Can be Sold`: enabled
+- :guilabel:`Product Type`: select :guilabel:`Service`
+- :guilabel:`Invoicing Policy`: select :guilabel:`Prepaid/Fixed Price`
+- :guilabel:`Create on Order`: select :guilabel:`Nothing`
+- :guilabel:`Sales Price`: enter the amount of the top-up
+
+.. note::
+   In order to have eWallet top-ups of different amounts, create multiple top-up products and
+   modify the :guilabel:`Sales Price` accordingly.
+
+Once the top-up is created, go to :menuselection:`Sales app --> Products --> Gift cards & eWallet`
+to :guilabel:`Create` an eWallet program. The following configuration options are available:
+
+- :guilabel:`Program Name`: enter a name for the eWallet program
+- :guilabel:`Program Type`: select :guilabel:`eWallet`
+- :guilabel:`eWallet Products`: select the eWallet top-up created earlier. Repeat the process if
+  you created top-ups of different amounts.
+- :guilabel:`Email template`: select the email template used for the email sent to the customer. To
+  create a new template, click on the field, select :guilabel:`Search More`, and then click
+  :guilabel:`Create`.
+- :guilabel:`Currency`: select the currency to use for the eWallet program
+- :guilabel:`Company`: select the company for which the program is valid and available
+- :guilabel:`Available On`: select the applications on which the program is valid and available
+- :guilabel:`Website`: select the website on which the program is valid and available. Leave this
+  field empty to include all websites.
+- :guilabel:`Point of Sale`: select the :abbr:`PoS (Point of Sale)` in which the program is valid
+  and available. Leave this field empty to include all :abbr:`PoS (Point of Sale)`.
+
+.. image:: ewallets_giftcards/ewallet-configuration.png
```

Comment:
[Compressing](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#images) this image reduces file size by ~3x.